### PR TITLE
cmake: fix ccache detection

### PR DESCRIPTION
**Pull request by perfect-daemon**

### CODE REVIEW

File: cmake/FindCcache.cmake [hunk 1]
```diff
@@ -42,12 +42,19 @@
...
+	set(TEST_PROJECT "${CMAKE_BINARY_DIR}/${CMAKE_FILES_DIRECTORY}/CMakeTmp")
+	file(WRITE "${TEST_PROJECT}/CMakeLists.txt" [=[
+cmake_minimum_required(VERSION 3.1)
+project(test)
+option (CCACHE "")
+file(WRITE "${CMAKE_SOURCE_DIR}/test.cpp" "int main() { return 0; }")
+set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE}")
+set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK    "${CCACHE}")
+add_executable(test test.cpp)
+]=])
```
Comment (mj-xmr) [inline cmake/FindCcache.cmake:54]: Please tabulate this block twice.

**Review by mj-xmr — CHANGES_REQUESTED** [review]

**Comment by mj-xmr** [comment]
I have checked the logs. It appears to work properly. Everything works like before (of course FreeBSD's ccache is still failing, like before)

File: cmake/FindCcache.cmake (hunk 1 above)
Comment (perfect-daemon) [reply cmake/FindCcache.cmake:54]:
```
Bracket argument can't be formatted due to limitations[0] of CMake language without affecting result.
[0] https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#bracket-argument
```

**Review by selsta — APPROVED** [review]
Fixes ccache detection on my Mac.

File: cmake/FindCcache.cmake (hunk 1 above)
Comment (mj-xmr) [reply cmake/FindCcache.cmake:54]: OK then.

**Review by mj-xmr — APPROVED** [review]

**Review by selsta — APPROVED** [review]

**Pull request merged**